Add section to changes on triggering tasks at icp and fcp using regex… (#876)
cylc/cylc-doc: src/reference/changes.rst

For more information on Cylc IDs, run ``cylc help id``.

> Triggering tasks at initial and final cycle point

Reviewer: This isn't just a trigger feature, suggest a more abstract title, maybe:

```suggestion
Selecting tasks at initial and final cycle point
```

?

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The initial and final cycle points in a workflow can now be referenced in
commands using the ``^`` and ``$`` symbols, e.g:

> ```shell
> $ cylc trigger myworkflow//^/mytask
> $ cylc trigger myworkflow//$/mytask
> ```

Reviewer: Could do with an explanation of the example commands. Would be good to use a command other than trigger to set expectations.

```suggestion
```bash
# trigger "mytask" at the initial cycle point
cylc trigger myworkflow//^/mytask

# set required outputs for "mytask" at the final cycle point
cylc set myworkflow//$/mytask

# remove all tasks at the final cycle point
cylc remove myworkflow//$
```
```

For more information on Cylc IDs, run ``cylc help id``.

Selecting tasks at initial and final cycle point
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The initial and final cycle points in a workflow can now be referenced in
commands using the ``^`` and ``$`` symbols, e.g:

> ```bash
> # trigger "mytask" at the initial cycle point
> cylc trigger myworkflow//^/mytask
> 
> # set required outputs for "mytask" at the final cycle point
> cylc set myworkflow//$/mytask
> 
> # remove all tasks at the final cycle point
> cylc remove myworkflow//$

Author: ```suggestion
.. code-block:: console

   # trigger "mytask" at the initial cycle point
   cylc trigger myworkflow//^/mytask

   # set required outputs for "mytask" at the final cycle point
   cylc set myworkflow//$/mytask

   # remove all tasks at the final cycle point
   cylc remove myworkflow//$
```
